ios/Lunes/Info.plist

```diff
@@ -44,6 +44,8 @@
 	</dict>
 	<key>NSCameraUsageDescription</key>
 	<string>Zum Scannen von QR-Codes benötigt LUNES Zugriff auf deine Kamera.</string>
+	<key>NSMicrophoneUsageDescription</key>
+    <string>Zur Aufnahme von Audio Files benötigt LUNES Zugriff auf dein Mikrofon</string>
```

Comment:
Maybe "Zur Aufnahme von Audios benötigt LUNES Zugriff auf dein Mikrofon". Finde das "Audio Files" irgendwie komisch.

Maybe you can also adjust the string for camera usage, as this is not only for QR-Codes any longer and I forgot to adjust this.